packages/datadog-esbuild/src/utils.js

```diff
@@ -7,9 +7,12 @@ const path = require('node:path')
 const { NODE_MAJOR, NODE_MINOR } = require('../../../version.js')
 
 const getExportsImporting = (url) => import(url).then(Object.keys)
-const getExports = NODE_MAJOR >= 20 || (NODE_MAJOR === 18 && NODE_MINOR >= 19)
-  ? require('import-in-the-middle/lib/get-exports.js')
-  : getExportsImporting
+
+const isModernNode = NODE_MAJOR >= 20 || (NODE_MAJOR === 18 && NODE_MINOR >= 19)
```

Comment:
Could you check if this is still needed when loading iitm with ESM? I am not sure what was introduced at that version (the special handling could use a comment).